johnsonandjohnson/Bodiless-JS
# feat(core-ui): add a warning element for user warning messages.

## Changes
- Add and use a warning form element for in the pull changes panel when conflict exists

## Test Instructions
- Create a conflict that cannot be pulled.
- Click pull changes.
- Expect to see a warning as described in the issue #416 


## Related Issues
- Closes #416
